tracer/src/Datadog.Trace/DataStreamsMonitoring/SpanExtensions.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="SpanExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.DataStreamsMonitoring;
+
+internal static class SpanExtensions
```

Comment:
nit: new files should have `#nullable enable`

```suggestion

#nullable enable

namespace Datadog.Trace.DataStreamsMonitoring;

internal static class SpanExtensions
```